sources/packages/backend/apps/load-test-gateway/src/main.ts

```diff
@@ -0,0 +1,14 @@
+import "../../../env-setup";
+import { NestFactory } from "@nestjs/core";
+import { LoadTestModule } from "./load-test.module";
+import { ConfigService } from "@sims/utilities/config";
+import { KeycloakConfig } from "@sims/auth/config";
+
+async function bootstrap() {
+  await KeycloakConfig.load();
+  const app = await NestFactory.create(LoadTestModule);
```

Comment:
For my understanding, `NestFactory.create<NestExpressApplication>` was not used because there was not benefit in this case?

Reply:
Yes, I felt specifying a type would not have any benefit here. Let me know if you think otherwise.